pkg/auth/userinfo.go

```diff
@@ -60,14 +57,14 @@ func NewUserInfoFromContext(ctx context.Context) (*UserInfo, bool) {
 func NewPublicUser() *UserInfo {
```

Comment:
Do you want to change this API name also?

Reply:
Yep, nice catch. Fixed this